src/olympia/scanners/models.py

```diff
@@ -295,6 +304,30 @@ class ScannerQueryRule(AbstractScannerRule):
     class Meta(AbstractScannerRule.Meta):
         db_table = 'scanners_query_rules'
 
+    def change_state_to(self, target):
```

Comment:
Fairly basic but makes it more clear what states are and what they can be changed to.

Reply:
I like it.